package.json

```diff
@@ -50,7 +50,6 @@
     "ember-load-initializers": "^0.5.1",
     "ember-lodash": "0.0.11",
     "ember-resolver": "^2.0.3",
-    "ember-welcome-page": "^2.0.1",
```

Comment:
This package added some unwanted stylesheets to the index page. The package only creates the default Ember index page, so we don't need it.